xml/System.Runtime.Intrinsics/Vector128.xml

```diff
@@ -124,11 +128,13 @@
         <Parameter Name="vector" Type="System.Runtime.Intrinsics.Vector128&lt;T&gt;" RefType="this" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="vector">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of the input vector.</typeparam>
+        <param name="vector">The vector to reinterpret.</param>
+        <summary>Reinterprets a <see cref="T:System.Runtime.Intrinsics.Vector128`1" /> as a new <see cref="T:System.Runtime.Intrinsics.Vector128" /> of type <see cref="T:System.Double" />.</summary>
```

Comment:
Hey @tannergooding, this is the workaround I had to use so we could properly show the non-generic type for each method. Does it still make sense? Would this work for you?
CC @rpetrusha

Reply:
Apparently I missed a few descriptions. I'll address them.